hgt/getHGTElevation.js

```diff
@@ -0,0 +1,47 @@
+// This file is largely derived from https://github.com/perliedman/node-hgt/blob/master/src/hgt.js
+module.exports = function getHGTElevation(hgt, coord) {
+    const size = hgt.size - 1;
+    const row = (coord[1] - hgt.swLngLat[1]) * size;
+    const col = (coord[0] - hgt.swLngLat[0]) * size;
+
+    if (row < 0 || col < 0 || row > size || col > size) {
+        throw new Error(
+            'Latitude/longitude is outside tile bounds (row=' +
```

Comment:
nit: use template literal instead of +. E.g., `blah blah (row=${row}, col=${col}; blah blah`